### PR TITLE
Command Line Interface implementation

### PR DESCRIPTION
**Pull request by xavierlmendez**

### CODE REVIEW

**Review by thomaskahng — APPROVED** [review]
Approved

File: src/dbms/userInterface/CLI.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    }
+
+    public void getInput(){
+        System.out.print("kyptonDB> ");
+        input = ins.nextLine();
+    }
+
+    public void processInput(){
+        List<String> commands = List.of(input.split("/[;]/"));
+        Stream<Object> resultMessages = commands.stream().map(commandService::execute);
```
Comment (haofast) [inline src/dbms/userInterface/CLI.java:30]: would stream type of `Stream<String>` work here?

File: src/dbms/userInterface/commandService.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            return "invalid syntax";
+        }
+        return processCommand(commandArgs);
+    }
+
+    public static String processCommand(List<String> commandArgs){
+        CommandType commandType = CommandType.EXIT;
+        String firstKey = commandArgs.get(0);
+        String secondKey = commandArgs.get(1);
+        if(firstKey.compareTo("SHOW") == 0){  }
```
[the remark this answers is not in the document]
Comment (xavierlmendez) [reply src/dbms/userInterface/commandService.java:20]: This should be a rudimentary keyword parser for the types of commands we can allow. This takes advantage of the first two keywords of a command being unique for all the commands we're implementing per the spec given

File: src/dbms/userInterface/commandService.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        else if (firstKey.compareTo("DROP") + secondKey.compareTo("TABLE") == 0) {
+            commandType = CommandType.DROP_TABLE;
+        }
+        else if (firstKey.compareTo("DROP") + secondKey.compareTo("INDEX") == 0) {
+            commandType = CommandType.DROP_INDEX;
+        }
+        else if (firstKey.compareTo("INSERT") == 0) {commandType = CommandType.INSERT;}
+        else if (firstKey.compareTo("DELETE") == 0) {commandType = CommandType.DELETE;}
+        else if (firstKey.compareTo("UPDATE") == 0) {commandType = CommandType.UPDATE;}
+        else if (firstKey.compareTo("SELECT") == 0) {commandType = CommandType.SELECT;}
```
[the remark this answers is not in the document]
Comment (xavierlmendez) [reply src/dbms/userInterface/commandService.java:36]: Just used the base linter on ide to implement this

**Review by thomaskahng — APPROVED** [review]

**Pull request merged**